src/api/put-product.ts

```diff
@@ -4,22 +4,28 @@ import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
 import { Product } from "../model/product";
 import { DynamoDbStore } from "../store/dynamodb/dynamodb-store";
 import { ProductStore } from "../store/product-store";
+import { captureLambdaHandler } from '@aws-lambda-powertools/tracer';
+import { injectLambdaContext } from '@aws-lambda-powertools/logger';
+import {logMetrics, MetricUnits} from '@aws-lambda-powertools/metrics';
+import middy from "@middy/core";
+import {logger, metrics, tracer} from "../powertools/utilities";
 
 const store: ProductStore = new DynamoDbStore();
-export const handler = async (
+const lambdaHandler = async (
   event: APIGatewayProxyEvent
 ): Promise<APIGatewayProxyResult> => {
-  console.info(`Event body ${event.body}`);
+
   const id = event.pathParameters!.id;
   if (id === undefined) {
-    console.warn("Missing 'id' parameter in path");
+    logger.warn('[PUT products] Missing \'id\' parameter in path');
```

Comment:
This should probably be `[PUT product]` (singular)

Reply:
Well spotted!